src/section.js

```diff
@@ -51,26 +51,55 @@ class Section {
 		var request = _request || this.request || Request;
 		var loading = new defer();
 		var loaded = loading.promise;
-		if (this.contents) {
-			loading.resolve(this.contents);
-		} else {
-			//KEM: this is where the file is loaded and turned into xml
-			//KEM: add in a load to the smil file and append it?
-			request(this.url)
-				.then(function (xml) {
-					// var directory = new Url(this.url).directory;
 
```

Comment:
The main change in section is that instead of loading just the page url, I also load the smil file first and then feed the result into where it loads the main page, in order to append it onto the iframe html.